code/modules/mob/living/simple_animal/abnormality/waw/hatred_queen.dm

```diff
@@ -156,6 +159,8 @@
 				TryTeleport()
 			if(5)
 				if(datum_reference?.qliphoth_meter == 2) //only able to use normal if passive
+					if(prob(5))
+						addtimer(CALLBACK(src, .atom/movable/proc/say, "With love!"))
```

Comment:
No. Don't force player controlled mobs to say stuff, it will annoy them.

Reply:
Just add a !client check